.env.oss

```diff
@@ -7,19 +7,21 @@
 # ------------------------
 # OSS Environment
 # ------------------------
-APP_TIMEOUT=29s
+ADMIN_URL=http://admin-staging.artsy.net
 API_REQUEST_TIMEOUT=10000
-PORT=5000
+API_URL=https://stagingapi.artsy.net
+APP_TIMEOUT=29s
 CLIENT_ID=e750db60ac506978fc70
 CLIENT_SECRET=3a33d2085cbd1176153f99781bbce7c6
-METAPHYSICS_ENDPOINT=https://metaphysics-staging.artsy.net
-API_URL=https://stagingapi.artsy.net
-POSITRON_URL=https://stagingwriter.artsy.net
-PREDICTION_URL=https://live-staging.artsy.net
-ADMIN_URL=http://admin-staging.artsy.net
 CMS_URL=http://cms-staging.artsy.net
-WEBFONT_URL=http://webfonts.artsy.net
+ENABLE_GRAPHQL_PROXY=true
 GEMINI_ACCOUNT_KEY=force
 GEMINI_CLOUDFRONT_URL=https://d7hftxdivxxvm.cloudfront.net
 GENOME_URL=https://helix.artsy.net
+METAPHYSICS_ENDPOINT=https://metaphysics-staging.artsy.net
+PORT=5000
```

Comment:
Acceptance tests are complaining about nothing listening on `4000`. Does this need an update?

Reply:
Lets see. it keeps hanging, but runs fine locally.

Reply:
oh yay, that was indeed it. ty 👍